static/js/admin.js

```diff
@@ -110,6 +127,15 @@ exports.setup_page = function () {
         error: failed_listing_streams
     });
 
+    // Populate emoji table
+    channel.get({
+        url: '/json/realm/emoji',
+        idempotent: true,
+        timeout: 10 * 1000,
+        success: populate_emoji,
+        error: failed_listing_emoji
+    });
```

Comment:
I think we don't need this query -- the realm emoji are already sent down to the browser client for all users (admin or no) in `page_params.realm_emoji` and processed in static/js/emoji.js (and updated as they are changed by the `realm_emoji` event in server_events.js).